ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs

```diff
@@ -88,6 +102,18 @@ internal class BiggestGridDetectionPcuStrategy : AbstractGridDetectionStrategy {
                     continue;
 
                 gridsList.Add(cubeGrid);
+                if (config.ExcludeProjectionPCU)
+                {
```

Comment:
This looks like Code Duplikation to me, could you extract it into its own method so we only have that once.

Also does the BlockCount Strategy have the same issue as the PCU strategy?

Reply:
Moved it into the CountProjectionPCU method, for the BlockCount Strategy it doesn't count projected blocks towards the topgrid.